corehq/apps/toggle_ui/views.py

```diff
@@ -149,6 +164,43 @@ def post(self, request, *args, **kwargs):
             clear_toggle_cache(toggle.slug, item)
 
         data = {
-            'item_list': item_list
+            'items': item_list
         }
+        if self.usage_info:
+            data['last_used'] = _get_usage_info(toggle)
         return HttpResponse(json.dumps(data), content_type="application/json")
+
+
+def _get_usage_info(toggle):
+    DATE_FORMAT = "%Y-%m-%d"
+    last_used = {}
+    for enabled in toggle.enabled_users:
+        name = _enabled_item_name(enabled)
+        try:
+            if _get_namespace(enabled) == NAMESPACE_DOMAIN:
+                last_used[name] = Domain.get_by_name(name).last_modified.strftime(DATE_FORMAT)
+            else:
+                last_used[name] = CouchUser.get_by_username(name).last_login.strftime(DATE_FORMAT)
+        except (ResourceNotFound, AttributeError):
```

Comment:
`AttributeError` is a really generic exception that could easily mask a fat-finger coding mistake. I assume you're catching that in case `.last_modified` or `.last_login` are `None`. Consider putting the date formatting in a function that explicitly checks for `None`:
```py
def format_date(value):
    if value is None:
        return "Not Found"
    return value.strftime(DATE_FORMAT)

...
        try:
            if _get_namespace(enabled) == NAMESPACE_DOMAIN:
                last_used[name] = format_date(Domain.get_by_name(name).last_modified)
            else:
                last_used[name] = format_date(CouchUser.get_by_username(name).last_login)
        except ResourceNotFound:
```